src/components/ItaliaTheme/Header/HeaderSearch/SearchModal.jsx

```diff
@@ -338,10 +337,9 @@ const SearchModal = ({ closeModal, show }) => {
               )}
               className="ml-auto btn btn-outline-primary text-capitalize"
               style={{ visibility: advancedSearch ? 'visible' : 'hidden' }}
-              aria-label={intl.formatMessage(messages.search)}
               onClick={submitSearch}
             >
-              {intl.formatMessage(messages.confirmSearch)}
```

Comment:
Se ho visto bene, questo id non è più usato ma è rimasto il message all'inizio del file, farei pulizia in caso.